File: 1064.py
lista=[]
for i in range(6):
    num=float(input())
    if num > 0:
        lista.append(num)
soma=sum(lista)
positivos=len(lista)
media=soma/len(lista)
print(f"{positivos} valores positivos")
print(f"{media:.1f}")

File: cpp/1064.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){

    int cont= 0;
    double soma= 0;

    for(int i=0; i<6; i++){
        double num; cin >> num;
        if (num >= 0){
            cont++;
            soma+=num;
        }
    }

    cout << cont << " valores positivos" << "\n";
    cout << fixed << setprecision(1) << soma/cont << "\n";


}